nsis/README.txt

```diff
@@ -1,7 +1,7 @@
-This builds a one-click install for Vim for Win32 using the Nullsoft
+This builds a one-click install for Vim for OS Windows using the Nullsoft
```

Comment:
I've never seen the wording "OS Windows" in other part of Vim document.
"MS-Windows" might be better?

Reply:
Yes, it would be better that way. MS Windows.